addons/mail/static/src/core/web/chatter.js

```diff
@@ -328,11 +357,18 @@ export class Chatter extends Component {
         this.toggleComposer();
         this.state.jumpThreadPresent++;
         // Load new messages to fetch potential new messages from other users (useful due to lack of auto-sync in chatter).
-        this.load(this.props.threadId, ["followers", "messages", "suggestedRecipients"]);
+        this.load(this.props.threadModel, this.props.threadId, [
+            "followers",
+            "messages",
+            "suggestedRecipients",
+        ]);
     }
 
     onAddFollowers() {
-        this.load(this.state.thread.id, ["followers", "suggestedRecipients"]);
+        this.load(this.state.thread.model, this.state.thread.id, [
```

Comment:
Why are some using props and other using state? It looks like we should be consistent here.